### ISSUE TITLE
Logstash 6.1.3 NoMethodError

### ISSUE THREAD

**Issue by arko-r** [elastic/logstash#9081]
Logstash crashes when indexing linestrings:

~~~
`Failed to execute action {:action=>LogStash::PipelineAction::Stop/pipeline_id:main, :exception=>"NoMethodError", :message=>"undefined method `map' for nil:NilClass\nDid you mean?  tap", :backtrace=>["E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/util.rb:40:in `thread_info'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/pipeline.rb:670:in `block in plugin_threads_info'", "org/jruby/RubyArray.java:2486:in `map'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/pipeline.rb:670:in `plugin_threads_info'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/pipeline_reporter.rb:66:in `block in to_hash'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/util/wrapped_synchronous_queue.rb:80:in `inflight_batches'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/pipeline_reporter.rb:56:in `to_hash'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/pipeline_reporter.rb:51:in `snapshot'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/shutdown_watcher.rb:88:in `pipeline_report_snapshot'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/shutdown_watcher.rb:63:in `block in start'", "E:/Program/logstash/logstash-6.1.3/vendor/bundle/jruby/2.3.0/gems/stud-0.0.23/lib/stud/interval.rb:20:in `interval'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/shutdown_watcher.rb:59:in `start'", "E:/Program/logstash/logstash-6.1.3/logstash-core/lib/logstash/shutdown_watcher.rb:35:in `block in start'"]}`
~~~

The input is a MSSQL server using jdbc and the output is Elasticsearch.
There are no other errors visible when using the TRACE loglevel.
All the data is send to and received by Elasticsearch, but Logstash still gives this error.

- Version: 6.1.3
- Operating System: Windows 10

**Comment by andrewvc** [comment]
@arko-r we think we've fixed this in the latest Logstash release. Can you give it a shot and let us know?

**Comment by original-brownbear** [comment]
Closing here, this looks fixed in #9097.
